eap73-openj9-image-stream.json

```diff
@@ -0,0 +1,135 @@
+            "openshift.io/provider-display-name": "Red Hat, Inc."
+        }
+    },
+    "items": [
+        {
+            "kind": "ImageStream",
+            "apiVersion": "v1",
+            "metadata": {
+                "name": "jboss-eap73-openj9-openshift",
+                "annotations": {
+                    "openshift.io/display-name": "JBoss EAP 7.3 with OpenJDK11 + OpenJ9",
+                    "openshift.io/provider-display-name": "Red Hat, Inc.",
+                    "version": "7.3"
+                }
+            },
+            "labels": {
+                "xpaas": "7.3"
+            },
+            "spec": {
+                "tags": [
+                    {
+                        "name": "latest",
+                        "annotations": {
+                            "description": "The latest available build of JBoss EAP 7.3 with OpenJDK 11 + OpenJ9",
+                            "iconClass": "icon-eap",
+                            "tags": "builder,eap,javaee,java,jboss,hidden",
+                            "supports": "eap:7.3,javaee:8,java:11",
+                            "sampleRepo": "https://github.com/jbossas/eap-quickstarts/openshift",
+                            "sampleContextDir": "kitchensink",
+                            "sampleRef": "7.3.x-openshift",
+                            "version": "latest",
+                            "openshift.io/display-name": "JBoss EAP 7.3 with OpenJDK11 + OpenJ9 (Latest)"
+                        },
+                        "referencePolicy": {
+                            "type": "Local"
+                        },
+                        "from": {
+                            "kind": "DockerImage",
+                            "name": "registry.redhat.io/jboss-eap-7/eap73-openj9-11-openshift-rhel8:latest"
+                        }
+                    },
+                    {
+                        "name": "7.3",
+                        "annotations": {
+                            "description": "The latest available build of JBoss EAP 7.3 with OpenJDK 11 + OpenJ9",
+                            "iconClass": "icon-eap",
+                            "tags": "builder,eap,javaee,java,jboss,hidden",
+                            "supports": "eap:7.3,javaee:8,java:11",
+                            "sampleRepo": "https://github.com/jbossas/eap-quickstarts/openshift",
+                            "sampleContextDir": "kitchensink",
+                            "sampleRef": "7.3.x-openshift",
+                            "version": "latest",
+                            "openshift.io/display-name": "JBoss EAP 7.3 with OpenJDK11 + OpenJ9"
+                        },
+                        "referencePolicy": {
+                            "type": "Local"
+                        },
+                        "from": {
+                            "kind": "DockerImage",
+                            "name": "registry.redhat.io/jboss-eap-7/eap73-openj9-11-openshift-rhel8:7.3.0"
```

Comment:
should the `7.3` image stream tag be mapped to the docker image `:7.3` rather than `:7.3.0`?

Reply:
What we've done in the past is create the imagestream tag of 7.3 but then mapped that to the :latest image in RHCC, I think thats probably the best approach here too. We'd then have 7.3.0 tag that would be the latest build of 7.3.0, eg it could be 7.3.0-13 etc.